doc/user_guide/configuration.rst

```diff
@@ -181,3 +181,45 @@ iteration-spaces        This contains definitions of additional iteration spaces
                         in the :ref:`gocean1.0-configuration` section of the
                         GOcean1.0 chapter.
 ======================= =======================================================
+
+``NEMO`` Section
+^^^^^^^^^^^^^^^^^^^^^
+This section contains configuration options that are only applicable when
+using the NEMO API.
+
+.. tabularcolumns:: |l|L|
+
+======================= =======================================================
+Entry                   Description
+======================= =======================================================
+mapping-TYPE            This declares a mapping for a certain loop level,
+                        specified as TYPE. Each value must have three key:value
```

Comment:
Please also document the fact that TYPE will be the "type" of any corresponding Loop objects in the PSyIR - as returned by `Loop.loop_type`

Reply:
I've extended this section somewhat, especially added more comments to the example below. Hope that's good now.

Reply:
Nice, thanks.